### PR TITLE
Gateway doc update

### PR DESCRIPTION
**Pull request by luke-song**
Work in progress with getting started and tutorial of gateway protocol v2

### CODE REVIEW

File: docs/getting-started/Gateway Protocol.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 2
+---
+
+# How do I start with Gateway Protocl after I installed it?
```
Comment (JulianSpring) [inline docs/getting-started/Gateway Protocol.md]:
```suggestion
# How do I start with Gateway Protocol after I installed it?
```

File: docs/getting-started/Gateway Protocol.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+We have two options once we have installed Gateway Protocol.
+
+1. use of OFF-CHAIN GATEKEEPER: This is a Standard Use Case of creating a Network and having an EXTERNAL Gatekeeper as trusted issuer.
+
+Let's say a particular dApp is in needs to operate KYC checks of users to sell them a particular product, such as alcohol. 
+
+Trusted party such as Socure as a off-chain gatekeeper can provide KYC, Bot Protection, Carbon Utilization, etc. For their services, they can charge a fee to a dApp. When dAPP uses pass operation, Gatekeeper draw funds from dApp's Top-Up Wallet.
```
Comment (JulianSpring) [inline docs/getting-started/Gateway Protocol.md]:
```suggestion
Trusted party such as Socure as an off-chain gatekeeper can provide KYC, Bot Protection, Carbon Utilization, etc. For their services, they can charge a fee to a dApp. When dAPP uses pass operation, Gatekeeper draw funds from dApp's Top-Up Wallet.
```

File: docs/gateway-protocol/overview/overview.md [hunk 3]
```diff
@@ -9,3 +9,12 @@ The Gateway Protocol is a cross-chain oracle protocol that allows decentralized
 The identity verification process is completed by one or more Gatekeepers in a Gatekeeper Network. The Gatekeepers enforce the predetermined requirements and issue a Gateway Pass once the user successfully completes the verification.
 
 The Gateway Pass then allows the user to interact with the dApp without any further interaction without further verification prompts.
+
+# High-Level Motivation
+
+In a soon to be regulated world, dApps will need to comply with KYC/AML and other identity requirements. This is a huge burden for dApps to take on, and it is not in their best interest to do so. Hence, the gateway protocol was built.
```
Comment (DiosAkhilleus) [inline docs/gateway-protocol/overview/overview.md]: I'd capitalize "gateway protocol" here to be "Gateway Protocol"

File: docs/getting-started/Gateway Protocol.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+sidebar_position: 2
+---
+
+# How do I start with Gateway Protocol after I installed it?
+
+We have two options once we have installed Gateway Protocol.
+
+1. use of OFF-CHAIN GATEKEEPER: This is a Standard Use Case of creating a Network and having an EXTERNAL Gatekeeper as trusted issuer.
+
+Let's say a particular dApp is in needs to operate KYC checks of users to sell them a particular product, such as alcohol.
```
Comment (DiosAkhilleus) [inline docs/getting-started/Gateway Protocol.md]: Change "is in needs" to just be "needs"

File: docs/getting-started/Gateway Protocol.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+# How do I start with Gateway Protocol after I installed it?
+
+We have two options once we have installed Gateway Protocol.
+
+1. use of OFF-CHAIN GATEKEEPER: This is a Standard Use Case of creating a Network and having an EXTERNAL Gatekeeper as trusted issuer.
+
+Let's say a particular dApp is in needs to operate KYC checks of users to sell them a particular product, such as alcohol. 
+
+Trusted party such as Socure as an off-chain gatekeeper can provide KYC, Bot Protection, Carbon Utilization, etc. For their services, they can charge a fee to a dApp. When dAPP uses pass operation, Gatekeeper draw funds from dApp's Top-Up Wallet.
```
Comment (DiosAkhilleus) [inline docs/getting-started/Gateway Protocol.md]: "A trusted party"

File: docs/getting-started/Gateway Protocol.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+Socure Demo: https://zerodayshopping.identity.com
+
+Every time a user wants to use the dApp, they need to pay the fee to the off-chain gatekeeper. The off-chain gatekeeper will then issue a token to the user. The user can then use the token to access the dApp. The off-chain gatekeeper can also revoke the token if the user is not using the dApp anymore.
+
+Each Gatekeeper network will publish and maintain its own network compensation table. The table specifies the prices (for Gatekeepers) and fees (for the network/protocol) associated with each Pass operation within the network.
+
+2. use of ON-CHAIN GATEKEEPER: Here on-chain program is the Gatekeeper / Gatekeeper Network
```
Comment (DiosAkhilleus) [inline docs/getting-started/Gateway Protocol.md]: "Here an on-chain program..."

**Review by JulianSpring — APPROVED** [review]

**Pull request merged**